### PR TITLE
Changelog for new blueprints

### PR DESCRIPTION
**Pull request by MarianRaphael**
## Description

Changelog entry for the new blueprints

## Related Issue(s)

https://github.com/FlowFuse/CloudProject/issues/328#issuecomment-1903690603

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/changelog/2024/01/new-blueprints.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+We added new blueprints to FlowFuse Cloud. They are available on FlowFuse Cloud. Additionally, upon request, all our Teams and Enterprise Self-Hosted customers gain access to this collection.
+
+List of new Blueprints:
+- Andon Live
+- CRUD for Mongo DB
+- 5S Manufacturing checklist
```
Comment (ZJvandeWeg) [inline src/changelog/2024/01/new-blueprints.md:13]:
```suggestion
- [5S Manufacturing checklist](/blueprints/manufacturing/5s-checklist/)
```

File: src/changelog/2024/01/new-blueprints.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+title: New Blueprints added
+date: 2024-01-22 13:00:00.0
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+We added new blueprints to FlowFuse Cloud. They are available on FlowFuse Cloud. Additionally, upon request, all our Teams and Enterprise Self-Hosted customers gain access to this collection.
+
+List of new Blueprints:
+- Andon Live
```
Comment (ZJvandeWeg) [inline src/changelog/2024/01/new-blueprints.md:11]:
```suggestion
- [Andon Live](/blueprints/manufacturing/andon-live/)
```

File: src/changelog/2024/01/new-blueprints.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+- CRUD for Mongo DB
```
Comment (ZJvandeWeg) [inline src/changelog/2024/01/new-blueprints.md:12]:
```suggestion
- [CRUD for Mongo DB](/blueprints/getting-started/crud/)
```

File: src/changelog/2024/01/new-blueprints.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+- 5S Manufacturing checklist
+- Performance Operator Terminal
```
Comment (ZJvandeWeg) [inline src/changelog/2024/01/new-blueprints.md:14]:
```suggestion
- [Performance Operator Terminal](/blueprints/manufacturing/performance-overview/)
```

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/changelog/2024/01/new-blueprints.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+title: New Blueprints added
+date: 2024-01-22 13:00:00.0
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+We added new blueprints to FlowFuse Cloud. They are available on FlowFuse Cloud. Additionally, upon request, all our Teams and Enterprise Self-Hosted customers gain access to this collection.
```
Comment (knolleary) [inline src/changelog/2024/01/new-blueprints.md:8]:
```suggestion
We have added some new blueprints to FlowFuse Cloud to help users get started with a range of common manufacturing scenarios.

They are available to all users on FlowFuse Cloud. Additionally, upon request, all our Teams and Enterprise Self-Hosted customers gain access to this collection.
```

File: src/changelog/2024/01/new-blueprints.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+They are available to all users on FlowFuse Cloud. Additionally, upon request, all our Teams and Enterprise Self-Hosted customers gain access to this collection.
+
+List of new Blueprints:
+- [Andon Live](/blueprints/manufacturing/andon-live/)
+- [CRUD for Mongo DB](/blueprints/getting-started/crud/)
+- [5S Manufacturing checklist](/blueprints/manufacturing/5s-checklist/)
+- [Performance Operator Terminal](/blueprints/manufacturing/performance-overview/)
+
+Curious about what Blueprints are? [Click here](https://flowfuse.com/blog/2023/10/blueprints/)
+For an overview of all available Blueprints, [click here](https://flowfuse.com/blueprints/)
```
Comment (knolleary) [inline src/changelog/2024/01/new-blueprints.md:19]:
```suggestion
Curious about what Blueprints are? Read our recent blog post that [introduced Blueprints to the FlowFuse](https://flowfuse.com/blog/2023/10/blueprints/).

For an overview of all available Blueprints, check out our [Blueprint Library](https://flowfuse.com/blueprints/).
```

Making the link text more google friendly

**Review by knolleary — APPROVED** [review]

**Comment by knolleary** [comment]
This is good to merge as soon as we have the 👍🏻 from @ZJvandeWeg

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**